pkg/web/transactions.go

```diff
@@ -440,10 +441,13 @@ func (s *Server) SendEnvelopeForTransaction(c *gin.Context) {
 		return
 	}
 
+	detailURL, _ := url.JoinPath("/transactions", transaction.ID.String(), "info")
+	// Set a cookie to show a toast message on the page redirect.
+	setToastCookie(c, "transaction_send_success", "true", detailURL, s.conf.Auth.CookieDomain)
+
```

Comment:
None of the recommend ways to do this worked with htmx, likely because of the redirect. After attempting to use a flash message library or gin's sessions library to display a flash message, this felt like the best option since we hopefully shouldn't have to do this elsewhere.